src/makeCalculator.js

```diff
@@ -37,7 +37,33 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    add(number) {
+      calculator.result += number;
+    },
+    subtract(number) {
+      calculator.result -= number;
+    },
+    multiply(number) {
+      calculator.result *= number;
+    },
+    divide(number) {
+      calculator.result /= number;
+    },
+    operate(callback, num) {
+      callback(num);
+
+      return this;
+    },
+    reset() {
+      calculator.result = 0;
+
+      return this;
+    },
+    result: 0,
```

Comment:
You could use closure and have this result inside the body function, not in the object.